README.md

```diff
@@ -318,7 +318,9 @@ To learn more about the Nightscout API, visit https://YOUR-SITE.com/api-docs.htm
   An option plugin to enable adding foods from database in Bolus Wizard and enable .
 
 ##### `rawbg` (Raw BG)
-  Calculates BG using sensor and calibration records from and displays an alternate BG values and noise levels.
+  Calculates BG using sensor and calibration records from and displays an alternate BG values and noise levels. Defaults that can be adjusted with [extended setting](#extended-settings)
```

Comment:
can you elaborate on what the difference is between the default behaviour (both are false), and if either `SMOOTH` or `UNFILTERED` is true. Shouldn't be documented that one of these settings must set to be true (and not both)?

Reply:
Can you elaborate on what setting you would advice in what situation?